src/platform/user/tests/unauthed-flows.e2e.spec.js

```diff
@@ -6,7 +6,7 @@ module.exports = E2eHelpers.createE2eTest(client => {
   const appPaths = [
     // While the page is in maintenance, it doesn't need authed
     '/education/gi-bill/post-9-11/ch-33-benefit/status',
-    '/health-care/health-records',
+    '/health-care/get-medical-records',
```

Comment:
This will also be reverted. This was a weird case, because the VA redirect points to a content page, and the app itself shouldn't be directly linked to. So, the link replacement isn't 1:1.